index.html

```diff
@@ -1,277 +1,334 @@
-        <button type="button" class="card-button">See Project</button>
-      </div>
-    </section>
-     <section class="work-cards">
-      <div class="card">
-        <h3>Profesional Art Printing Data</h3>
-        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-        <ul class="card-ul">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>ruby</li>
-        </ul>
-        <button type="button" class="card-button">See Project</button>
-      </div>
-    </section>
-    <section class="work-cards">
-      <div class="card">
-        <h3>Profesional Art Printing Data</h3>
-        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-        <ul class="card-ul">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>ruby</li>
-        </ul>
-        <button type="button" class="card-button">See Project</button>
-      </div>
-    </section>
-    <section class="work-cards">
-      <div class="card">
-        <h3>Profesional Art Printing Data</h3>
-        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-        <ul class="card-ul">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>ruby</li>
-        </ul>
-        <button type="button" class="card-button">See Project</button>
-      </div>
-    </section> 
-    <section class="work-cards">
-      <div class="card">
-        <h3>Profesional Art Printing Data</h3>
-        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-        <ul class="card-ul">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>ruby</li>
-        </ul>
-        <button type="button" class="card-button">See Project</button>
-      </div>
-    </section> 
+    <!-- headline-container end -->
+
+
+      <!-- My Recent Works section start  -->
+      <section class="work">
+        <div class="work-header">
+          
+          <h2 class="work-title">My Recent Works</h2>
+          <div class="work-header-img"></div>
+          <img src="images/work-title-img.svg" alt="">
+        </div>
+        <div class="work-body">
+         <img src="#" alt="">
+          <article class="work-article">
+             <h2>Multi-Post Stories</h2>
+             <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has text been the industry's standard dummy ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+             <ul class="work-links">
+              <li><a href="#">Html</a></li>
+              <li><a href="#">Css</a></li>
+              <li><a href="#">Javascript</a></li>
+              <li><a href="#">Ruby</a></li>
```

Comment:
The tags and button also seem aligned on the right a little bit.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/221350311-b7a912ed-448f-4470-8ccc-583dd3b89e55.png)| ![image](https://user-images.githubusercontent.com/61361037/221350299-eceea995-acc7-4d05-b6b9-de3156bd2598.png)|